### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by vext01**
Update JSON structure, and talk a bit about auditing system services.

### CODE REVIEW

**Comment by snim2** [comment]
Fixes #302

File: README.md [hunk 1]
```diff
@@ -176,7 +176,62 @@ $ pwd
 $ make java-bench
 ```
 
-## Step 5: Run the example experiment
+## Step 5: Audit system services
+
+You should take some time to review the services running on your benchmarking
+machine. Debian especially has a habit of starting daemons which get pulled in
+by dependencies.
```
Comment (snim2) [inline README.md]: This sort of advice is good to have!

File: README.md [hunk 2]
```diff
@@ -176,7 +176,62 @@ $ pwd
...
+Note that by default Debian machines do not use a service like ntpd to set the
+system time. Instead the time is set using `ntpdate` when a network interface
+comes up.
+
+### Linux
+
+On Linux, list services with:
+
+```
+# systemctl | grep running
```
Comment (snim2) [inline README.md]: I think we used `$` for the system prompt below

**Comment by snim2** [comment]
Minor comments, but this looks good

File: README.md (hunk 2 above)
Comment (vext01) [inline README.md]: A hash is used to show it has to be run by root.

File: README.md [hunk 3]
```diff
@@ -209,23 +264,21 @@ The structure of the JSON results is as follows:
...
+    'core_cycle_counts': {      # Per-core core cycle counter deltas
+        'bmark:VM:variant': [
+            [                   # One list per process execution
+                [...], ...      # One list per core
+            ]
+    },
+    'aperf_counts': {...}       # Per-core APERF deltas
+                                # (structure same as 'core_cycle_counts')
+    'mperf_counts': {...}       # Per-core MPERF deltas
+                                # (structure same as 'core_cycle_counts')
```
Comment (vext01) [inline README.md]: Probably somewhere else, but that place doesn't exist. It's true that Krun docs are not complete, but I don't think now is the time to launch into that.

File: README.md (hunk 2 above)
Comment (snim2) [inline README.md]: OK, I think this is the only place we used a hash prompt, and where we used `sudo`, we didn't have a prompt at all!

File: README.md (hunk 3 above)
Comment (snim2) [inline README.md]: OK

**Comment by vext01** [comment]
Assuming you are happy with my responses, OK to squash?

**Comment by snim2** [comment]
Can you just look at line 122 and make it like your other root prompts? It should be:

```
# useradd krun
```

not:

```
sudo useradd krun
```

to match the changes you want to merge here.

**Comment by vext01** [comment]
Sure, fixed.

**Comment by snim2** [comment]
Great, let's squash then!

**Comment by vext01** [comment]
splat.

**Pull request merged**